Nikita_Konkov_Application/CSharpException/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using CSharpException;
+
+Console.WriteLine("Enter three sides of triangle:");
+Triangle triangle = new Triangle();
+
+int firstSide = int.Parse(Console.ReadLine());
+int secondSide = int.Parse(Console.ReadLine());
+int thirdSide = int.Parse(Console.ReadLine());
+do
+{
+    try
+    {
+        triangle.FirstSide = firstSide;
+
+        triangle.FirstSide = secondSide;
+
+        triangle.FirstSide = thirdSide;
+    }
+
+    catch (UnsuitableZeroValue ex)
+    {
+        Console.WriteLine(ex.Message);
+       
+        triangle.ZeroCheck(firstSide);
+        triangle.ZeroCheck(secondSide);
+        triangle.ZeroCheck(thirdSide);
+    }
+
+} while (triangle.IsZero(firstSide) || triangle.IsZero(secondSide) || triangle.IsZero(thirdSide));
```

Comment:
you should make up another condition for loop because the program throws an exception and exits.